db/migrate/20170206222252_add_win_option_to_bet_options.rb

```diff
@@ -0,0 +1,5 @@
+class AddWinOptionToBetOptions < ActiveRecord::Migration[5.0]
+  def change
+    add_column(:bet_options, :win_option, :boolean)
```

Comment:
I would prefer `:winner` over `:win_option`. Might just be a personal preference though